src/geo/bboxUtils.ts

```diff
@@ -15,28 +15,31 @@ const snapMinCordToTileGrid = (cord: number, tileRes: number): number => {
  * @returns bbox that contains the original bbox and match tile grid lines
  */
 export const snapBBoxToTileGrid = (bbox: BBox2d, zoomLevel: number): BBox2d => {
+  const initValue = -9999;
```

Comment:
why?
if you want to receive a valid BBOX - throw exception for invalid input

Reply:
done